### PR TITLE
Javier_Hemerson Administration Trail Project CR

### PR DESCRIPTION
**Pull request by miftah1991**
This is the Project of my Coding partner and I want to give a CR on this Project

### CODE REVIEW

**Comment by miftah1991** [comment]
Hi @miftah1991 

Good job!
You are almost there but there are some issues that you have to work on it.
Required Changes 
Check the comments under the review.

I strongly recommend these comments to consider it as they will make your code professional.

Keep it up 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by miftah1991** [comment]
Please make sure to add a screenshot of the project in readme file

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (miftah1991) [inline index.html:3]: Please make sure the file you included is existed as there is no file available by name of style.css

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+ 
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (miftah1991) [inline index.html:10]: Please always try to put the script and style tag in html head section

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+
```
Comment (miftah1991) [inline index.html:25]: Please make sure to configure linter in your project for more information please check the [linter documentation page](https://github.com/microverseinc/linters-config).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  <body id="main">
+    <header>
+      <section id="title">
+        <h1>Dr. Norman Borlaug</h1>
+        <h3>The man who saved a billion lives</h3>
+        <div id="img-div">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id="image" / alt = "image Dr working">
+          <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields</p>
+        </div>
```
Comment (Deepakdanger) [inline index.html:35]: This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @miftah1991, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] Let's use the GitHub inline comment. See the example inline comment review below.
- [ ] I think your comments are not clear enough. Giving clear feedback helps with effective communication, instead of asking to make a change only also add "why" the changes are necessary. See this article on [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more details.
- [ ] Also, it's recommended to write at least 2 comments in the review.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comment under one single summary like this current review. :wink:

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by miftah1991** [comment]
Hi @Deepakdanger ,

Thank you for your kind review, sure I will consider your comments in my CR 

Thank you 🙂

**Review by miftah1991 — COMMENTED** [review]
Hi @Deepakdanger ,

Thank you for your kind review, sure I will consider your comments in my CR

Thank you 🙂

**Review by Lordkaito — APPROVED** [review]
Hi @miftah1991, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] [OPTIONAL] Although you did a good job, I highly recommend to give explanatory comments in any review. This is good to help the other person fully understand what you are requesting and why. For example, instead of saying "Please make your Readme.md file more descriptive" you can say "Please, make sure to use a descriptive Readme.md file, with no empty sections. This is good because this file is the first impression to any other developer, hiring manager or person that sees the project, and will help that person to understand the purpose and use of the project".

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by miftah1991** [comment]
@Lordkaito Thank you for your kind review.

**Pull request merged**